src/components/ShoppingList/ListView.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useCollection } from 'react-firebase-hooks/firestore';
+import { db } from '../../lib/firebase';
+
+const ListView = () => {
+  const [value] = useCollection(db.collection(localStorage.getItem('token')), {
+    snapshotListenOptions: { includeMetadataChanges: true },
```

Comment:
As we talked about on the call, it seems there's an opportunity for refactor since we're calling this same code as we did in the parent component. We can consider passing the 'value' down through props to this component.